File: Problemset/number-of-good-pairs/number-of-good-pairs.php

// @Title: 好数对的数目 (Number of Good Pairs)
// @Author: 846188037@qq.com
// @Date: 2020-07-13 00:06:00
// @Runtime: 12 ms
// @Memory: 14.3 MB

class Solution {

    /**
     * @param Integer[] $nums
     * @return Integer
     */
    function numIdenticalPairs($nums) {
        $count=0;
        for($i=0;$i<count($nums)-1;$i++){
            for($j=$i+1;$j<count($nums);$j++){
                if($nums[$i]==$nums[$j]){
                    $count++;
                }
            }
        }
        return $count;

    }
}

File: Problemset/number-of-good-pairs/number-of-good-pairs.cpp

// @Title: 好数对的数目 (Number of Good Pairs)
// @Author: 846188037@qq.com
// @Date: 2020-07-13 01:21:57
// @Runtime: 0 ms
// @Memory: 7.3 MB

class Solution {
public:
    int numIdenticalPairs(vector<int>& nums) {
        int count=0;
        for(int i=0;i<nums.size()-1;i++){
            for(int j=i+1;j<nums.size();j++){
                if(nums[i]==nums[j]){
                    count++;
                }
            }
        }
        return count;

    }
};
